ext/devcontainer/src/azd/devcontainer-feature.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "azd",
+    "version": "1.0.0",
+    "name": "Azure Developer CLI",
+    "documentationURL": "https://github.com/azure/azure-dev/tree/main/ext/devcontainer/src/azd",
+    "description": "Installs the Azure Developer CLI along with needed dependencies.",
+    "options": {
+      "version": {
+        "type": "string",
+        "proposals": [
+          "latest"
+        ],
+        "default": "latest",
+        "description": "Select or enter an Azure Developer CLI version. (Available versions may vary by Linux distribution.)"
+      }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+              "ms-azuretools.vscode-bicep",
+              "ms-vscode.vscode-node-azure-pack"
```

Comment:
```suggestion
              "ms-vscode.vscode-node-azure-pack",
              "ms-azuretools.azure-dev"
```

Reply:
Does this look like the right set of extensions to add, or should we just have `ms-azuretools.azure-dev`?